docs/orchestration/faq/upgrading_1.0.md

```diff
@@ -0,0 +1,165 @@
+# Upgrading to Prefect 1.0
+
+Prefect 1.0 includes some important changes that may require updates to your flow and task definitions. 
+
+Most significant: support has been removed for some features previously marked as deprecated. This means flows using deprecated features may encounter errors rather than warnings.
+
+The following sections describe changes you should be aware of for making sure your flows work as expected with Prefect 1.0.
+
+[[toc]]
+
+## API keys replace authentication tokens
+
+[API keys](/orchestration/concepts/api_keys.html) replace authentication tokens to authenticate users and service accounts with the Prefect Cloud API. Existing authentication tokens will be ignored.
+
+See [Removing API tokens](/orchestration/concepts/api_keys.html#removing-api-tokens) to learn how to remove old tokens from your environment. See [Using API keys with older versions of Prefect](/orchestration/concepts/api_keys.html#using-api-keys-with-older-versions-of-prefect) for details about how you can use API keys in place of tokens in certain situations (such as a VM or container built with an older version of Prefect).
+
+The Prefect CLI commands `create-token`, `revoke-token`, and `list-tokens` have been removed.
+
+The `prefect auth login` and `prefect auth logout` commands now use API keys. Previously, logging out with an API token would just reset your tenant and access token, but leave the API token on disk. Now, we retain that behavior the first time the command is called, but if you call it a second time, we delete the API token. This allows users to easily remove their API token in favor of an API key.
```

Comment:
Note you begin mixing "API token" and "authentication token". Generally, I've tried to use "API key" and "authentication token" to further separate the concepts.

Reply:
I think even just "API key" and "token" would be good.